.github/workflows/log_management.yml

```diff
@@ -12,7 +12,10 @@ jobs:
     steps:
       - uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938
       - name: Workflow Housekeeper - workflows NOT in default branch
+
         uses: JosiahSiegel/workflow-housekeeper@731cc20bb613208b34efb6ac74aab4ba147abb50
+        ## emvaldes@hotmail.com (Replace) - uses: ./.github/actions/workflow-housekeeper
```

Comment:
corp email please

Reply:
I have updated the email address to reflect the Aquia corporate email.